### PR TITLE
Company API views

### PR DESCRIPTION
**Pull request by monikasulik**
This PR is against the ED-145 branch for easier reviewing. Will need to be closed and then reopened against master once #4 is merged.

### CODE REVIEW

File: company/views.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from company.models import Company
+
+
+class CompanyRetrieveUpdateAPIView(RetrieveUpdateAPIView):
+
+    model = Company
+    serializer_class = CompanySerializer
+
+    def get_queryset(self):
+        return self.model.objects.all()
```
Comment (richtier) [inline company/views.py]: as a follow up PR we can add a permission class that checks if the logged in user has access to the company.

File: company/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    assert response.json() == expected
+
+
+@pytest.mark.django_db
+def test_company_update_view():
+    client = APIClient()
+    company = Company.objects.create(
+        number='12345678', aims=json.dumps(['AIM101']))
+
+    response = client.put(
```
Comment (richtier) [inline company/tests/test_views.py]: as discussed, please also add a test for `patch`

File: company/views.py (hunk 1 above)
Comment (konradko) [inline company/views.py]: ye let's not worry about permissions yet

**Comment by konradko** [comment]
@monikasulik can this be merged?

**Comment by monikasulik** [comment]
Nope, not yet but soon.

**Comment by monikasulik** [comment]
Closing this to reopen against master.

**Comment by konradko** [comment]
ah sorry, didn't realise this is against another branch

**Pull request closed without merging**